### PR TITLE
add FinalAppointmentsController specs

### PR DESCRIPTION
**Pull request by ericcf**

### CODE REVIEW

File: spec/controllers/api/dialogues_controller_spec.rb [hunk 1]
```diff
@@ -2,5 +2,6 @@
 
 module Api
   RSpec.describe DialoguesController, type: :controller do
+    fixtures :all
```
Comment (eschlange) [inline spec/controllers/api/dialogues_controller_spec.rb]: Is the addition of fixtures into the controller spec files a preparatory step for the addition of specs?

Comment (ericcf) [inline spec/controllers/api/dialogues_controller_spec.rb]: Yes, just a placeholder until I get to the specs!

File: spec/controllers/final_appointments_controller_spec.rb [hunk 2]
```diff
@@ -1,4 +1,196 @@
...
+    {
+      appointment_at: Time.new, appointment_location: "l", phone_returned: true
+    }
+  end
+
+  let(:invalid_final_appointment_params) do
+    { appointment_at: nil, appointment_location: nil, phone_returned: nil }
+  end
+
+  shared_examples "a bad request" do
```
Comment (eschlange) [inline spec/controllers/final_appointments_controller_spec.rb]: This is a very convenient utility! I need to use this more often...

**Comment by eschlange** [comment]
Looks good.

**Pull request merged**